apollo-client/src/main/java/com/ctrip/framework/apollo/internals/ConcurrentConfigManager.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.internals;
+
+import com.ctrip.framework.apollo.Config;
+import com.ctrip.framework.apollo.ConfigFile;
+import com.ctrip.framework.apollo.core.enums.ConfigFileFormat;
+import com.google.common.collect.Maps;
+
+import java.util.Map;
+
+/**
+ * @author tian
+ */
+public class ConcurrentConfigManager extends DefaultConfigManager {
```

Comment:
Why don't we integrate the lock key logic into the DefaultConfigManager? I think the updated logic will be activated anyway. Do we have specific reasons to move it to the ConcurrentConfigManager?

Reply:
It seems that expanding the concurrency capability from DefaultConfigManager is not necessary. It should be enhanced on the original class, and I will refactor according to the suggestions.